site/components/layout/Socials.js

```diff
@@ -0,0 +1,15 @@
+import SocialIconLink from './SocialIconLink'
+
+const Socials = () => (
+  <div className="flex gap-3">
+    <SocialIconLink href="https://x.com/hemi_xyz" iconName="twitter" />
+    <SocialIconLink href="https://discord.gg/hemixyz" iconName="discord" />
+    <SocialIconLink href="https://github.com/hemilabs" iconName="github" />
+    <SocialIconLink
+      href="https://www.linkedin.com/company/hemi-labs"
+      iconName="linkedin"
+    />
```

Comment:
@rmilrad  should we add Youtube? I think it is on all the other pages (it's in the portal and the main website).

Similar question about Telegram links (though it is not in the portal)

Reply:
This kind of change shouldn't be required on another task?
I mean, the PR is correct according the Issue and its approved layout.

Reply:
@rafappelt sure, it can be done in another PR, I'm not against it. But I'm just asking Ryan 😄  Thomas can feel free to merge the PR and create an issue if desired

Reply:
The PR can only be merged if @gabmontes (code owner) approves